### PR TITLE
ISPN-16213 Test endpoint it root list strings

### PR DESCRIPTION
**Pull request by fax4ever**
https://issues.redhat.com/browse/ISPN-16213
https://issues.redhat.com/browse/JDG-7111

Based on https://github.com/infinispan/protostream/pull/284 => draft

### CODE REVIEW

File: build/configuration/pom.xml [hunk 1]
```diff
@@ -192,7 +192,7 @@
       <version.ow2.asm>9.7</version.ow2.asm>
       <version.picketbox>5.0.3.Final</version.picketbox>
       <version.picketlink>2.5.5.SP12</version.picketlink>
-      <version.protostream>5.0.5.Final</version.protostream>
+      <version.protostream>5.0.6-SNAPSHOT</version.protostream>
```
Comment (tristantarrant) [inline build/configuration/pom.xml:195]: Can you rebase this PR to pick up ProtoStream 5.0.7.Final which contains your fix ?

**Review by tristantarrant — CHANGES_REQUESTED** [review]

File: build/configuration/pom.xml (hunk 1 above)
Comment (fax4ever) [reply build/configuration/pom.xml:195]: Sure!

**Comment by fax4ever** [comment]
Pushed, sorry for the delay

**Comment by pruivo** [comment]
merged! thanks @fax4ever !

**Comment by fax4ever** [comment]
Thanks Pedro!

**Pull request merged**